swagger.yaml

```diff
@@ -35,6 +35,8 @@ paths:
     post:
       tags:
         - installer
+      security:
+        - userAuth: [admin, user]
```

Comment:
Since most APIs are permitted for `[read-only-admin, user]` I suggest that this be the default and for the others define it per API
The default is defined in line 31 above